src/main/java/com/mathworks/ci/MatlabBuild.java

```diff
@@ -99,8 +99,9 @@ default void prepareTmpFldr(FilePath tmpFldr, String runnerScript) throws IOExce
         zipFileLocation.unzip(tmpFldr);

```

Comment:
Something to think about for the future, would be pretty clean if we could swing only copying the zip file into the .matlab folder and then unzipping it into the containers local temp folder. This way, there never is any "m-files" that show up in the workspace last all and could get mistakenly used in code coverage.

Reply:
Sure , Tracking it with issue #205